src/integration_tests/constants.go

```diff
@@ -12,17 +14,48 @@ const (
 	GENESIS_FILE  = TEST_DATA_FOLDER + "/topology/genesis_ledger.json"
 	TOPOLOGY_FILE = TEST_DATA_FOLDER + "/topology/topology.json"
 
-	APP_CONFIG_FILE = TEST_DATA_FOLDER + "/topology/uptime_service_config/app_config.json"
-	AWS_CREDS_FILE  = TEST_DATA_FOLDER + "/topology/uptime_service_config/aws_creds.json"
-	MINASHEETS_FILE = TEST_DATA_FOLDER + "/topology/uptime_service_config/minasheets.json"
+	UPTIME_SERVICE_CONFIG_DIR = TEST_DATA_FOLDER + "/topology/uptime_service_config"
+	APP_CONFIG_FILE           = UPTIME_SERVICE_CONFIG_DIR + "/app_config.json"
 
 	TIMEOUT_IN_S = 600
 )
 
-var uptime_service_config_files = []string{
-	APP_CONFIG_FILE,
-	AWS_CREDS_FILE,
-	MINASHEETS_FILE,
+func getDirFiles(dir string, suffix string) ([]string, error) {
+	files, err := os.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	var gpgFiles []string
+	for _, f := range files {
+		if strings.HasSuffix(f.Name(), suffix) {
+			absolutePath := dir + "/" + f.Name()
+			gpgFiles = append(gpgFiles, absolutePath)
+		}
+	}
+
+	return gpgFiles, nil
+}
```

Comment:
The variable in this function is named `gpgFiles` but is also used to get `json` files. A bit confusing. 
```suggestion
	var result []string
	for _, f := range files {
		if strings.HasSuffix(f.Name(), suffix) {
			absolutePath := dir + "/" + f.Name()
			result = append(result, absolutePath)
		}
	}

	return result, nil
}
```